### DISCUSSION TITLE
Error Updating iOS native dependencies

### DISCUSSION THREAD

**Discussion by kevinluk07** [ionic-team/capacitor#3521] [Q&A] [answered]
Hey all, I've created my first capacitor app in Android, and I'm trying to add iOS. I have been encountering this error, and if I ignore it and try to build in Xcode I've got the SIGBART error.

`npx cap sync ios
✔ Copying web assets from www to ios/App/public in 1.13s
✔ Copying native bridge in 20.19ms
✔ Copying capacitor.config.json in 1.11ms
✔ copy in 1.19s
✔ Updating iOS plugins in 5.50ms
  Found 0 Capacitor plugins for ios:
✖ Updating iOS native dependencies with "pod install" (may take several minutes): 
✖ update ios: 
[error] Error running update: Analyzing dependencies
Downloading dependencies
Generating Pods project
Integrating client project
Pod installation complete! There are 2 dependencies from the Podfile and 2 total pods installed.
xcode-select: error: tool 'xcodebuild' requires Xcode, but active developer directory '/Library/Developer/CommandLineTools' is a command line tools instance

Sync finished in 7.079s`

I've read that this xcode error could be ignored, so I'm not sure if it is related. Any idea? Thanks!

**Comment by dotNetkow** [comment, marked as answer]
The issue is here: _xcode-select: error: tool 'xcodebuild' requires Xcode, but active developer directory '/Library/Developer/CommandLineTools' is a command line tools instance_

[Here's some things](https://stackoverflow.com/questions/17980759/xcode-select-active-developer-directory-error) to try.

**Comment by kevinluk07** [comment]
Amazing! I swear I read somewhere that I could ignore this error. But selecting the Xcode app fixed the error in capacitor. I'm still having a SIGABRT error, but that would probably be my code.

Thanks!
